main.py

```diff
@@ -18,6 +18,16 @@ class bcolors:
 
 # Load the core functionality (help, on_ready, etc.)
```

Comment:
main.py and cog loading doesn't work this way anymore; I'm not really sure how to best reconcile the issue though, so let's ask others or do some research 😕